extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -2198,4 +2219,100 @@ Runnable updateCurrentAndLatestOffsets()
       }
     };
   }
+
+  /**
+   * Collect row ingestion stats from all tasks managed by this supervisor.
+   *
+   * @return A map of groupId->taskId->task row stats
+   *
+   * @throws InterruptedException
+   * @throws ExecutionException
+   * @throws TimeoutException
+   */
+  private Map<String, Object> getCurrentTotalStats() throws InterruptedException, ExecutionException, TimeoutException
+  {
+    Map<String, Object> allStats = Maps.newHashMap();
```

Comment:
`Map<String, Map<String, Object>>`?

Reply:
Changed this to Map<String, Map<String, Object>>, I had originally intended to leave the values as a more freeform object in case different supervisors have different implementations, but the method already forced the return to be a String->Object map, so I went ahead with the suggested change